tests/core/test_agent.py

```diff
@@ -74,6 +74,27 @@ async def test_training_data_is_reproducible():
         assert str(x.as_dialogue()) == str(same_training_data[i].as_dialogue())
 
 
+async def test_agent_train(trained_rasa_model: Text):
+    moodbot_domain = Domain.load("data/test_domains/default_with_slots.yml")
```

Comment:
as it's not moodbot anymore:
```suggestion
    domain = Domain.load("data/test_domains/default_with_slots.yml")
```